tests/queries/0_stateless/03321_system_tables_parametrized_view_params.sql

```diff
@@ -0,0 +1,21 @@
+DROP TABLE IF EXISTS raw_data;
+
+DROP TABLE IF EXISTS parametrized_view_one_param;
+
+DROP TABLE IF EXISTS parametrized_view_multiple_params;
+
+CREATE TABLE raw_data (id UInt32, data String) ENGINE = MergeTree ORDER BY id;
+
+CREATE VIEW parametrized_view_one_param AS SELECT * FROM raw_data WHERE id = {id:UInt32};
+
+SELECT name, engine, parametrized_view_paramters FROM system.tables WHERE name = 'parametrized_view_one_param';
```

Comment:
```suggestion
SELECT name, engine, parametrized_view_paramters FROM system.tables WHERE database = currentDatabase() and name = 'parametrized_view_one_param';
```

this will fix the style check as well

Reply:
It took me a long time to figure out why style check test is failed!!!
Thanks a lot for your help.